tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -249,6 +249,11 @@ CREATE TABLE IF NOT EXISTS asset_seedlings (
 
     asset_meta BLOB,
 
+    -- TODO(jhb): Redunant, could reference asset_groups? Need more guarantees.
+    -- Can be NULL if we enable emission but specify no group key.
+    -- Or use ZeroKey and then add length constraint?
+    group_key BLOB,
```

Comment:
Yeah we can do a joint based on the `genesis_id`. 

The path would be `genesis_id` -> `genesis_assets` -> `asset_groups`

That'll also naturally handle the requirement that we need the _plain_ group key here to make the proper proofs (see #91). 

Using a reference here means we don't also need to store this `group_key` several times for a set of assets that all are made in the same batch and share a group key.